mtoy.go

```diff
@@ -0,0 +1,51 @@
+package mtoy
+
+import (
+	"math/rand"
+	"time"
+)
+
+type mtoy struct {
+	answers []string
+}
+
+func New() *mtoy {
+	return &mtoy{
+		answers: []string{
```

Comment:
wondering if answers should be encapsulated or have it as a parameter to the New function.